docs/GraphQL.md

```diff
@@ -0,0 +1,755 @@
+# GUAC GraphQL definitions
```

Comment:
can we do a small section on what is GraphQL and link out to some graphQL getting started for those that are not familiar